server/build_event_protocol/event_parser/event_parser.go

```diff
@@ -315,6 +315,8 @@ func fillInvocationFromWorkspaceStatus(workspaceStatus *build_event_stream.Works
 			invocation.Host = item.Value
 		case "HOST":
 			invocation.Host = item.Value
+		case "PATTERN":
```

Comment:
Do we want the key to be something more specific, like `TARGET_PATTERN`? This matches the way that Bazel refers to them. And because this probably won't be the actual pattern, maybe `TARGET_PATTERN_OVERRIDE`?

Reply:
I'd rather match the BuildBuddy "pattern" naming that we use throughout than making it too Bazel specific: https://github.com/buildbuddy-io/buildbuddy/blob/bab9c19635c2d53c4e158c51a4e566a6691d34be/proto/api/v1/invocation.proto#L52

HOST and USER are also "overrides" and I'd rather not change those because it wouldn't be backwards compatible.